argus.js

```diff
@@ -12,8 +12,13 @@ const ArgusJS = function(token) {
     }, false);
 
     return {
-        token: () => currentToken,
-        get: (url) => fetch(url, { "method": "GET", "headers": { "Authorization": "Token " + currentToken } })
+        get: (url) => fetch(`${mdr_url}${url}`, { "method": "GET", "headers": { "Authorization": "Bearer " + currentToken.access } }),
```

Comment:
Can we have a function to join the base URL and the path? It should handle the URL having a leading '/' or not.

e.g.,

```
const mdrURL = '...';
const buildURL = (path) => (new URL(path, mdrURL).href;
```